--- filezilla-3.64.0/configure.ac.orig	2023-04-26 10:51:58.000000000 +0200
+++ filezilla-3.64.0/configure.ac	2023-06-20 19:56:00.275303589 +0200
@@ -595,11 +595,6 @@
 
   # We want xdg-open on *nix
   if echo "`$WX_CONFIG_WITH_ARGS --basename`" | grep -i gtk > /dev/null 2>&1; then
-    AC_PATH_PROG(xdgopen, xdg-open)
-    if test -z "$xdgopen"; then
-      AC_MSG_ERROR([xdg-open not found. This program is the preferred way to launch the default browser. Please install the xdg-utils.])
-    fi
-
     if echo "`$WX_CONFIG_WITH_ARGS --basename`" | grep -i gtk2 > /dev/null 2>&1; then
       PKG_CHECK_MODULES(LIBGTK, gtk+-2.0,, [
         AC_MSG_ERROR([gtk+-2.0 was not found, even though the used version of wxWidgets depends on it. Are you missing the gtk+2.0 development files?])
