--- filezilla-3.66.5/configure.ac.orig	2024-02-07 10:46:42.000000000 +0100
+++ filezilla-3.66.5/configure.ac	2024-03-05 22:57:48.058684746 +0100
@@ -594,11 +594,6 @@
 
     # We want xdg-open on *nix
     if echo "`$WX_CONFIG_WITH_ARGS --basename`" | grep -i gtk > /dev/null 2>&1; then
-      AC_PATH_PROG(xdgopen, xdg-open)
-      if test -z "$xdgopen"; then
-        AC_MSG_ERROR([xdg-open not found. This program is the preferred way to launch the default browser. Please install the xdg-utils.])
-      fi
-
       if echo "`$WX_CONFIG_WITH_ARGS --basename`" | grep -i gtk2 > /dev/null 2>&1; then
         PKG_CHECK_MODULES(LIBGTK, gtk+-2.0,, [
           AC_MSG_ERROR([gtk+-2.0 was not found, even though the used version of wxWidgets depends on it. Are you missing the gtk+2.0 development files?])
